README.md

```diff
@@ -161,3 +161,154 @@ const hawk = new HawkCatcher({
 
 hawk.connectVue(Vue)
 ```
+
+## Performance Monitoring
+
```

Comment:
- explain what is Transaction and Span.
- move API Reference on the first place
- optimization explanation is a technical detail, so it should be placed after usage description